rpcserver.go

```diff
@@ -6570,3 +6600,31 @@ func encodeVirtualPackets(packets []*tappsbt.VPacket) ([][]byte, error) {
 
 	return rawPackets, nil
 }
+
+// EncodePrecisionInJSON takes a precision value and an encoded json object and
+// attempts to append a specific key-value pair in the json object, then encodes
+// it again and returns it.
+func EncodePrecisionInJSON(precision uint64, jBytes []byte) ([]byte, error) {
```

Comment:
Could alternatively have this live in the `assets` package, or `taprpc`.

Reply:
moving to taprpc